Kudu.Core/Environment.cs

```diff
@@ -117,6 +117,13 @@ public class Environment : IEnvironment
             _dataPath = Path.Combine(rootPath, Constants.DataPath);
             _jobsDataPath = Path.Combine(_dataPath, Constants.JobsPath);
             _jobsBinariesPath = Path.Combine(_webRootPath, Constants.AppDataPath, Constants.JobsPath);
+            // if userDefinedWebJobRoot is relative path, append it to site/wwwroot/ else use it as an absolute path
```

Comment:
`_webRootPath = "D:/home/site/wwwroot/"`

1. default `_jobsBinariesPath = "D:/home/site/wwwroot/App_Data/jobs"`
2. if `userDefinedWebJobRoot = "myfunctions"`, `_jobsBinariesPath = "D:/home/site/wwwroot/myfunctions"`
3. if `userDefinedWebJobRoot = "D:/home/functionfolder"`, `_jobsBinariesPath = "D:/home/functionfolder"`

Path.Combine(path1,path2) returns path2 if path2 is an absolute path